<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Find &amp;what:</string>
     </property>
     <property name="buddy">
      <cstring>lineEditSearch</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditSearch"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="PushFind">
     <property name="text">
      <string>&amp;Find</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBoxCase">
     <property name="text">
      <string>Match &amp;case</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="PushClose">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBoxSearch">
     <property name="text">
      <string>Search &amp;backward</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>PushFind</sender>
   <signal>clicked()</signal>
   <receiver>FindDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PushClose</sender>
   <signal>clicked()</signal>
   <receiver>FindDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
